pipelines/data_embedding_pipeline/data_embedding_pipeline.py

```diff
@@ -15,19 +15,36 @@ def data_embedding_pipeline() -> None:
         load_data: A ZenML step which loads the data from a specified DVC data version.
         embed_data: A ZenML step which embeds the text data into vectors and pushes to the vector database.
     """
+    data_version = "data/first_version"  # should manually update this each time when dataset is collected.
```

Comment:
In my opinion, we will have to maintain the same `data_version` variable in two places. An alternate solution here would be to pass `data_version = "data/first_version"` to `data_embedding_pipeline` function and reuse it in all places. This way we have to change variable in one place avoids the chances to making mistake.

Reply:
After a discussion with Jon, we decided to include this in the config of load_data step and this step would also return two new variables, the current_data_version and the reference_data_version which would be used for the sub-sequence steps.

Reply:
Note: The same function is also being reused in `data_preperation_pipeline` [here](https://github.com/fuzzylabs/MindGPT/blob/e39291a77086406b6082fa1067be282f3fb303e7/pipelines/data_preparation_pipeline/data_preparation_pipeline.py#L21) and also needs to be updated.